### DISCUSSION TITLE
It wont let me download something.

### DISCUSSION THREAD

**Discussion by Cloud560** [infinitepower18/WSA-Sideloader#110] [Support] [answered]
It keeps saying error 10061

**Comment by infinitepower18** [comment, marked as answer]
This is a [bug](https://github.com/microsoft/WSA/issues/136) with the subsystem itself, restarting the PC will usually fix it.

If you still get this error, try these steps:

1. Make sure WSA is turned off and disable WSA autostart in Task Manager, startup apps before proceeding with the below steps

2. Disable Hyper-V using the command `dism.exe /Online /Disable-Feature:Microsoft-Hyper-V` and reboot your PC

3. Reserve port 58526 so Hyper-V doesn't reserve it back using the command `netsh int ipv4 add excludedportrange protocol=tcp startport=58526 numberofports=1`

4. Re-enable Hyper-V using the command `dism.exe /Online /Enable-Feature:Microsoft-Hyper-V /All` and reboot your PC

**Reply by 7pb1975-ai** [reply]
Thank you for your support, I followed all your instructions and windows subsystem finally displayed port 58526 as available for ADB to connect yet when I put the command ADB connect 127.0.0.1:58526 it disappears in the developer notes, I get an error 10061 but also it says there is no connection that target machine actively refuses it.  Please help if you can, thanks.
